ui-modules/blueprint-composer/app/components/catalog-selector/catalog-selector.less

```diff
@@ -314,6 +314,20 @@ catalog-selector {
             margin-top: 6px;
         }
     }
+    .quick-info-title-uforge {
```

Comment:
warning ?

Reply:
This one should stay "info", it is about the info pop, not the warning icon